aws-go-eks/go.mod

```diff
@@ -3,7 +3,7 @@ module aws-go-eks
 go 1.16
```

Comment:
Should we switch to 1.17? I guess pulumi-eks is still on 1.16 so this limits it?

Reply:
Yep, exactly.